### ISSUE TITLE
thefuck fails to install

### ISSUE THREAD

**Issue by patrickdepinguin** [fishery/fin#2]
```
$ fin install https://github.com/oh-my-fish/plugin-thefuck                                                                                                
fin SAYS Installing 1 plugin
         ✔  thefuck                 https://github.com/oh-my-fish/plugin-thefuck
fish: Unknown command '__tf_updt'
~/.config/fish/conf.d/init.init.fish (line 2):   set -q __tf_func; or __tf_updt
                                                                      ^
from sourcing file ~/.config/fish/conf.d/init.init.fish
	called on line 565 of file ~/.config/fish/functions/fin.fish

in function “__fin_plugin_enable”
	called on line 360 of file ~/.config/fish/functions/fin.fish
	with parameter list “~/.config/fin/thefuck”

in function “__fin_install”
	called on line 201 of file ~/.config/fish/functions/fin.fish
	with parameter list “https://github.com/oh-my-fish/plugin-thefuck”

in function “fin”
	called on standard input
	with parameter list “install https://github.com/oh-my-fish/plugin-thefuck”

fin SAYS Done in 1s 338ms
$ fin install https://github.com/oh-my-fish/plugin-thefuck                                              
fin WARN Nothing to install
```

I checked that fisherman can install thefuck, which is the case.

**Comment by bucaran** [comment]
Thanks, it was a problem with how I was walking the `init.fish` function to generate a unique name for `init.fish` files.

Note: You can run `fin omf/thefuck` too.

**Comment by bucaran** [comment]
@patrickdepinguin We have merged fin into fisherman and released it as 2.0 🎉 

Check it out https://github.com/fisherman/fisherman
